api/v1alpha1/driver_types.go

```diff
@@ -72,10 +72,6 @@ const (
 	// Disables the feature and remove the snapshotter sidercar
 	NoneSnapshotPolicy SnapshotPolicyType = "none"
 
-	// Inspect the CRD's for volumesnapshot and volumegroupsnapshot and enable
-	// corresponding features (will results in deployment of a snapshotter sidecar)
-	AutoDetectSnapshotPolicy SnapshotPolicyType = "autodetect"
```

Comment:
@nb-ohad now is it responsibility of the user to check if the CRDs exist and set the policy?

Reply:
Yes, but we might want to re-introduce this in the future.

The reason I dropped it for now is that it is more complicated than just looking for the CRD existence. We also need to make sure the images you are using support the feature, and report your detection on the status, maybe even with a reason. If we want the entire auto-detect mechinizem will be "magic" from a user perspective.